examples/terraform/vsphere_centos/cloud-config-metadata.tftpl

```diff
@@ -0,0 +1,9 @@
+instance-id: ${machine_name}
```

Comment:
TODO: add a comment somewhere to mention that users might need to adjust this file based on their vSphere environment.